### PR TITLE
Make the port column a little wider to stop truncation

### LINKED ISSUE #1711: details panel truncates port to four digits
ports can be five digits, so truncating at four is just mean

screenshot_2016-07-21_19-47-26

### PR DESCRIPTION
**Pull request by foot**
For firefox+linux which renders slightly tighter it seems

Fixes #1711

### CODE REVIEW

**Comment by rade** [comment]
replicasets and deployments need the same treatment. And hosts.

File: client/app/scripts/components/node-details/node-details-table.js [hunk 1]
```diff
@@ -27,11 +28,15 @@ const COLUMN_WIDTHS = {
...
   pid: CW.M,
-  port: '44px',
+  port: CW.S,
   ppid: CW.M,
   process_cpu_usage_percent: CW.M,
   process_memory_usage_bytes: CW.M,
   threads: CW.M,
+  //
+  // pods > services > details panel.
+  kubernetes_ip: '120px',
```
Comment (davkal) [inline client/app/scripts/components/node-details/node-details-table.js]: Why does this get a special value?

**Comment by davkal** [comment]
Bar comment, code LGTM, and looks good on Mac/Chrome

@rade could you confirm the port width fix viewed on Linux?

**Comment by rade** [comment]
> @rade could you confirm the port width fix viewed on Linux?

See https://github.com/weaveworks/scope/issues/1711#issuecomment-237305358 - i.e. I am no longer seeing that problem even without that change - looking at the images, it seems that the column width increased since I reported the problem.

**Comment by foot** [comment]
re: replicasets, hosts and deployments this fixes it for those too, bad commit message.

**Comment by rade** [comment]
> re: replicasets, hosts and deployments this fixes it for those too, bad commit message.

And comment in the code!

Are you sure this fixes *hosts*? What does that have to do with `kubernetes_ip`?

**Comment by foot** [comment]
Yes! and in the code.

Fixes hosts in the sense that the only place I could find an ip in the hosts panel was in the k8s section.

**Comment by rade** [comment]
> Fixes hosts in the sense that the only place I could find an ip in the hosts panel was in the k8s section.

Ah yes, good point. What an odd choice of columns - listing the IPs of pods but not containers.

**Comment by foot** [comment]
More long term solution could be covered in #1735

**Comment by foot** [comment]
Squashed

**Pull request merged**